pytestqt/plugin.py

```diff
@@ -414,6 +417,11 @@ def pytest_addoption(parser):
     parser.addini('qt_no_exception_capture',
                   'disable automatic exception capture')
 
+    parser.addoption('--no-qt-log', dest='qt_log', action='store_false',
+                     default=True)
+    parser.addoption('--qt-log-format', dest='qt_log_format',
+                     default='{rec.type_name}: {rec.message}')
```

Comment:
I was planning on add them as ini options too, but realized it is not really needed as the users can use `addopts` instead:

```ini
[pytest]
addopts = --qt-log-format="{rec.type_name}"
```

(I have to remember to add this to the documentation)

`qt_no_exception_capture`  is a config option because it is "part" of the configuration of the test suite, as it is not something the user should have to pass in the command line for the test suite to work. `python_files` follows the same idea, if a test suite uses `ut_` prefix for test files, it makes sense that this is configured "permanently" in the ini file, instead of having to be passed every time on the command line.

Hope my somewhat contrived explanation makes sense. :smile: